tests/console/consoletest_setup.pm

```diff
@@ -100,6 +103,12 @@ sub run() {
         assert_script_run("grub2-mkconfig -o /boot/grub2/grub.cfg");
     }
 
+    # https://trello.com/c/Y57ujlSn/663-3-plan-b-for-sle-12-sp2-p1-988157-hps-bug-sles12-sp2-install-gets-stuck-initializing-on-a-8-blade-hpe-bl920-gen-9-system
```

Comment:
having this link in our code is as useless - only yast devs can open it (and you)